### DISCUSSION TITLE
Custom 3D printed mount for Zenner water meter

### DISCUSSION THREAD

**Discussion by lkaino** [jomjol/AI-on-the-edge-device#2369] [General]
I made a custom holder for a Zenner water meter.

Please find the stl in the zip package.

[zenner_mount.zip](https://github.com/jomjol/AI-on-the-edge-device/files/11362061/zenner_mount.zip)

![WhatsApp Image 2023-04-30 at 21 28 08](https://user-images.githubusercontent.com/6444272/235370663-03ab2cd1-9a85-4d27-8060-c7c164e85eb9.jpeg)
![image](https://user-images.githubusercontent.com/6444272/235370642-2d849853-fd49-4695-bbb8-fa508f96b7e3.png)
![image](https://user-images.githubusercontent.com/6444272/235370624-0cc925e3-e2a4-487a-87e2-9ce0fcf2e3e3.png)
![image](https://user-images.githubusercontent.com/6444272/235370654-1e2e7422-79e5-46d6-95f1-f7a05f1b7aad.png)

**Comment by caco3** [comment]
Nicely done!

May I suggest to also provide the source files so others can adapt it?

**Reply by lkaino** [reply]
Good point @caco3!

Here's the zip file which contains the STEP file and fusion source files.
[zenner_mount_with_stl_step_f3z.zip](https://github.com/jomjol/AI-on-the-edge-device/files/11381057/zenner_mount_with_stl_step_f3z.zip)
